test/io/serializers/jsonHelpers/basicread.chpl

```diff
@@ -0,0 +1,11 @@
+use IO, JSON;
+
+record myPair : serializable {
+  var x: int;
+  var y: string;
+}
+
+var myValue = new myPair(1, "hello");
+var myValueJson = toJson(myValue);
```

Comment:
these values are unused, should they be removed from this test?

Reply:
Good catch, they should be.